app/scripts/directives/nav.js

```diff
@@ -1,25 +1,40 @@
 'use strict';
 
 angular.module('openshiftConsole')
-  .directive('sidebar', function(HawtioNav) {
+  .directive('sidebar', function($location, Constants) {
+    var itemMatchesPath = function(item, path) {
+      return (item.href === path) || _.some(item.prefixes, function(prefix) {
```

Comment:
I could strip the trailing slash.  But angular is doing that already, if you go to /overview/ it redirects you to overview/